app/controllers/sessions_controller.rb

```diff
@@ -26,5 +28,18 @@ def login_process(user)
     flash[:success] = "Welcome, #{user.name}!"
   end
 
-
+def new_user_id
+  if new_user = User.find_by(id: session[:user_id])
```

Comment:
finding the user within the session. If logic to make sure that there actually is a new user